### ISSUE TITLE
Explain both persistence and permanence in relation to IPFS.

### ISSUE THREAD

**Issue by dadepo** [ipfs/ipfs-docs#364]
The documentation page [Persistence, permanence and pinning](https://docs.ipfs.io/concepts/persistence/) has nothing about Persistence nor permanence, even though this is stated in the title. The content of the page is mainly about pinning which definitely relates to Persistence and permanence, but is not all about the topic. I am no expert here, but I will think permanence relates more to immutability, while Persistence relates to pinning. Also it will also be beneficial to explain the difference between Persistence and permanence.

In summary, this page is only about pinning, it will be more useful if it is expanded to actually talk about Persistence and permanence

**Comment by johnnymatthews** [comment]
Thanks for bringing this up @dadepo. You're point is valid, the title is misleading. The following tasks should be completed to fix this issue: 

- [ ] Briefly explain how IPFS handles data (this is already touched on by referring to IPFS as a _cache_ in the current version of this page).
- [ ] Explain what pinning is.
- [ ] Explain why pinning is important if a user wants to keep files available 
- [ ] Describe what pinning services are, and popular services currently available. 
- [ ] Link to the [`how-to/pin-files` guide](https://docs.ipfs.io/how-to/pin-files/#three-kinds-of-pins).

**Comment by dadepo** [comment]
Hi @johnnymatthews 

Allow me to offer a quick feedback on the tasks you highlighted...I still think the tasks you outlined are mostly relating to persistence and nothing about permanence.

I stumbled on that page in the documentation when I wanted to understand the difference between Permanence and Persistence, because although the two words sound like synonyms, they refer to slightly different concepts within IPFS.

For example I understand it is possible to have Permanence without Persistence. How is that possible? I think that is the kind of question this part of the documentation should help answer.

Hence I will suggest the following items to the ones you already outlined above:

- [ ] Explain what Permanence is, and how IPFS enables the Permanent Web
- [ ] Explain the relationship (or not) between Permanence and Persistence

**Comment by johnnymatthews** [comment]
That's excellent, thanks very much for your input here @dadepo! I've added your points into the above tasks. It's really helpful when we get in-depth feedback and suggestions from folks like yourself.

**Comment by johnnymatthews** [comment]
Just a heads up @dadepo, I've assigned a bounty to this issue. If you're comfortable completing it yourself and have some spare time on your hands, feel free to give it a go!

**Comment by bertrandfalguiere** [comment]
I would like to claim this if @dadepo won't

**Comment by dadepo** [comment]
@bertrandfalguiere please go ahead!

**Comment by johnnymatthews** [comment]
You read my mind @bertrandfalguiere! I was about to tag you in this issue to see if you'd like to tackle it. As always, leave questions and comments in this thread :)

**Comment by alexmmueller** [comment]
@bertrandfalguiere are you still working on this issue? If not, I would like to give it a shot.

**Comment by johnnymatthews** [comment]
Are you still working on this @bertrandfalguiere? If not, I'll unassigned you and open the floor to other contributors.

**Comment by alexmmueller** [comment]
@johnnymatthews I have started to create a draft for this issue. Maybe you want to assign it to me as @bertrandfalguiere has not yet answered?

**Comment by bertrandfalguiere** [comment]
Yes, sorry for letting you down. I won't be able to work on it anymore

**Comment by johnnymatthews** [comment]
No problem @bertrandfalguiere, life get it's the way sometimes :)

@alexmmueller, you're now assigned to this issue. Let me know if you have any questions.

**Comment by johnnymatthews** [comment]
Closed in #572.
